spec/features/visitors/password_recovery_spec.rb

```diff
@@ -226,43 +228,15 @@ def reset_password_and_sign_back_in(user, password = 'a really long password')
       end
     end
 
-    it 'displays error when password fields are empty & JS is on', js: true do
-      click_button t('forms.passwords.edit.buttons.submit')
-
-      expect(page).to have_content 'Please fill in this field.'
-    end
-
-    it 'displays field validation error when password fields are empty' do
-      click_button t('forms.passwords.edit.buttons.submit')
+    it 'does not allow the user to submit until password score is good', js: true do
+      fill_in t('forms.passwords.edit.labels.password'), with: 'invalid'
+      expect(page).not_to have_button(t('forms.passwords.edit.buttons.submit'))
 
-      expect(page).to have_content t('errors.messages.blank')
-    end
-
-    it 'displays field validation error when password field is too short' do
-      fill_in 'New password', with: '1234'
-      click_button t('forms.passwords.edit.buttons.submit')
-
-      expect(page).to have_content 'is too short (minimum is 8 characters)'
-    end
-
-    it "does not update the user's password when password is invalid" do
-      fill_in 'New password', with: '1234'
-      click_button t('forms.passwords.edit.buttons.submit')
-
-      signin(@user.email, '1234')
-      expect(current_path).to eq new_user_session_path
-    end
-
-    it 'allows multiple attempts with invalid password' do
-      fill_in 'New password', with: '1234'
-      click_button t('forms.passwords.edit.buttons.submit')
-
-      expect(page).to have_content 'is too short'
-
-      fill_in 'New password', with: '5678'
-      click_button t('forms.passwords.edit.buttons.submit')
+      fill_in t('forms.passwords.edit.labels.password'), with: 'password@132!'
+      expect(page).not_to have_button(t('forms.passwords.edit.buttons.submit'))
 
-      expect(page).to have_content 'is too short'
```

Comment:
Sometimes capybara cannot find an element with js turned on because there are overlapping elements. In that case, the solution is usually to trigger the click with js. See L119 in `spec/features/users/sign_in_spec.rb` for an example